apps/sensenet/cypress/integration/content-types/content-types.spec.ts

```diff
@@ -1,21 +1,21 @@
 import { pathWithQueryParams } from '../../../src/services/query-string-builder'
 
 describe('Content types', () => {
-  beforeEach(() => {
+  before(() => {
     cy.login()
     cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
       .get('[data-test="drawer-menu-item-content-types"]')
       .click()
-    cy.get('.ReactVirtualized__Table__Grid').scrollTo('bottom')
-    cy.xpath('//div[text()="Article"]').scrollIntoView({ duration: 500 }).as('articleRow')
   })
 
   it('clicking on the content types menu item should show article', () => {
-    cy.get('@articleRow').should('be.visible')
+    cy.get('.ReactVirtualized__Table__Grid').should('be.visible').scrollTo('bottom')
+    cy.get('[data-test="table-cell-article"]').as('article')
+    cy.get('@article').scrollIntoView().should('be.visible')
```

Comment:
cy.get('[data-test="table-cell-article"]').scrollIntoView().should('be.visible')